test/framework/src/main/java/org/elasticsearch/geo/XShapeTestUtil.java

```diff
@@ -15,11 +15,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
Don't we need to also update the license note here?

Reply:
No because it is a ASL2 license as it is a forked file from lucene. We are good here.